components/DepartmentsWalkthrough.vue

```diff
@@ -0,0 +1,297 @@
+            </v-row>
+          </v-col>
+          <v-col cols="1"></v-col>
+          <v-col>
+            <v-row>
+              <div class="Category-Slider">
+                <div class="Slider-Amount">${{department.storedValue}} mil</div>
+                <v-row justify="center">
+                  <v-slider
+                    v-bind:max="department.sliderMax"
+                    v-model="department.storedValue"
+                    min="0"
+                    vertical
+                    track-color="#B6DADA"
+                    color="#2A6465"
+                    @mousedown="tradeBtns"
+                  />
+                </v-row>
+              </div>
+            </v-row>
+            <v-row
+              class="Category-Budget-Caption text-sm-body-1"
+            >Last year’s budget: {{department.budget}}</v-row>
+          </v-col>
+        </v-row>
+      </div>
+    </transition-group>
+    <div class="carousel-controls">
+      <v-btn
+        v-for="(button) in buttonsShow"
+        v-bind:key="button.name"
+        class="carousel-controls__button"
+        v-bind:class="button.class"
+        v-bind:disabled="button.disabled"
+        v-bind:outlined="button.outlined"
+        rounded
+        depressed
+        width="240px"
+        color="#2A6465"
+        @click="navigate(button.text)"
+      >{{button.text}}</v-btn>
+    </div>
+    <div class="skip">SKIP TO OVERVIEW</div>
+  </v-container>
+</template>
+
+<script>
+import Vue from "vue";
+
+export default {
+  data() {
+    return {
+      index: 0,
+      departments: [
+        {
+          department: "community-health",
+          header: "Community Health",
+          content:
+            "The mission of the Department of Public Health is to protect and promote the health of all San Franciscans through the following divisions:<ul><li>San Francisco Health Network, which includes the Zuckerberg San Francisco General Hospital, Laguna Honda Hospital, ambulatory care, and transitions that oversee client flow throughout the system of care.</li><li>Population Health Division, which addresses public health concerns, including consumer safety, health promotion and disease prevention, and the monitoring of threats to the public’s health.</li></ul><br>Funds:<ul><li>Expand behavioral services for vulnerable residents, many of whom are experiencing homelessness</li><li>Provide department-wide workplace equity training and education</li><li>Implement a new electronic health record that supports clinical operations and revenue collections</li><li>Promote health education, physical activity, and healthy eating in communities with high rates of sugary drink consumption, diabetes, obesity, and heart disease</li><li>Increase staff at the hospitals and budget for other medical supplies including test kits for population health and prevention</li></ul>.",
+          budget: "$2,422 mil",
+          display: "inherit",
+          sliderMax: 16452,
```

Comment:
the fact that this is constant across departments indicates to me that it could be a top-level field in the data hash, e.g. a sibling of `index` and `departments`

Reply:
i actually subtract the previous values from each sliderMax as the user fills out the values per sophia's design so i think this is ok as is -- let me know if you disagree!